config/ksql-production-server.properties

```diff
@@ -18,17 +18,23 @@
 ### HTTP  ###
 # The URL the KSQL server will listen on:
 # The default is any IPv4 interface on the machine.
+# NOTE: If set to wildcard or loopback set 'inter.node.listener' to enable pull queries across machines
```

Comment:
https://kafka.apache.org/documentation/#configuration

We might want to be consistent with Kafka.  It seems like they use advertised.listeners.  Can we use a similar naming scheme?